src/Raven.Server/Documents/Indexes/IndexDefinitionBaseServerSide.cs

```diff
@@ -148,7 +148,7 @@ public static class IndexVersion
 
             public const long ProperlyParseThreeDigitsMillisecondsDates = 52_002; // RavenDB-17711
 
-            public const long EngineTypeStored = 54_000; // introducing Corax, added engine type to the index storage
+            public const long EngineTypeStored = 60_000; // introducing Corax, added engine type to the index storage
```

Comment:
It was introduced in 54_000 so it should stay like that. We shouldn't modify already defined version numbers.